src/pages/projectSettings/sharedFormSchema.ts

```diff
@@ -23,3 +23,15 @@ export const projectId = {
     "ui:optional": true,
   },
 };
+
+export const requestS3Creds = {
+  schema: {
+    type: "boolean" as "boolean",
+    title: "Request S3 Credentials",
```

Comment:
We could probably provide some more details here. Maybe a description that says something like "A JIRA ticket will be opened for the Build team enabling this project to read and write to an S3 bucket." Also, is there any means of providing users with the ticket number that is created?

Reply:
done

Reply:
oh also there is no point in providing users with the ticket because we currently do not do that 
and it is difficult to get the ticket because we create a notification that triggers a jira ticket creation